.circleci/config.yml

```diff
@@ -105,16 +110,13 @@ jobs:
           name: Linter
           command: yarn run tslint
   docs-deploy:
-    executor:
-      name: rn/macos
-      resource_class: macos.x86.medium.gen2
-      xcode_version: 14.3.1
+    <<: *base-mac-job
     steps:
       - checkout
       - install-dependencies:
           machine: "macos"
       - revenuecat/install-gem-mac-dependencies:
-          cache-version: v1
+          cache-version: v2
```

Comment:
Is this needed?

Reply:
I figured it wouldn't hurt since this is a new machine? that way it doesn't reuse the caches for the intel machine 🤷‍♂️